README.md

```diff
@@ -26,19 +26,20 @@ First, any packages required to complete the build are installed -
 this needs to be done for both the Debian environment and the specific
 board environment, which is why there are two lines.
 
-Next, the image is built.  Once the disk image is completed, it will placed
-in the `output` dir.
+The last command builds the image.  Once the disk image is completed,
+it will placed in the `output` dir.
 
 Using the image
 ---------------
 
-Once the image is built, and you have the disk image file from above.
-use "dd" (or similar) to write the image to the raw sdcard.
+Once the image is built, and you have the disk image file from above, write
+the image to the raw sdcard with a tool like [Etcher](https://etcher.io),
+or use `dd`:
```

Comment:
perhaps I'm old school, but I am making the assumption that dd is more useful for people who are running a build tool.

Either way, I think I would prefer leading with the tool that doesnt require you to install something else.

eg:

"... with a command line tool like dd (see below for example) or a graphical tool like Etcher ..."